ocean_lib/models/datatoken_base.py

```diff
@@ -287,16 +293,22 @@ def create_exchange(
         return (exchange, tx) if kwargs.get("full_info") else exchange
 
     @enforce_types
-    def get_exchanges(self) -> list:
+    def get_exchanges(self, active=True) -> list:
         """return List[OneExchange] - all the exchanges for this datatoken"""
         # import now, to avoid circular import
         from ocean_lib.models.fixed_rate_exchange import OneExchange
 
-        FRE = self._FRE()
+        # import now, to avoid circular import
+        from ocean_lib.models.fixed_rate_exchange import FixedRateExchange
+
+        exchanges = []
         addrs_and_exchange_ids = self.getFixedRates()
```

Comment:
I think there's a function that retrieves the objects directly, something like "get_exchanges" but I'm not sure about that. It could save some lines and clarity with instantiating everything